.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,21 @@
+---
+name: "\U0001F41B Bug report"
+about: Report a bug about the Scala compiler or the standard library
+title: ''
+assignees: ''
+
+---
+
+## minimized code
+
+```Scala
+println("hello, world")
+```
+
+## Compilation output
```

Comment:
it might not be a compile-time problem. maybe “## Results" or something

Reply:
Let's not be compiler-ist!  Semantics have bugs too, y'know?!